src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,93 @@
+import { useEffect, useMemo, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { PeopleFilters } from '../components/PeopleFilters';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { sortPeople } from '../utils/helpers';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isInitialized, setIsInitialized] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [searchParams] = useSearchParams();
+  const sort = searchParams.get('sort');
+  const order = searchParams.get('order');
+
+  const loadPeople = async () => {
+    setIsLoading(true);
+    setIsInitialized(false);
+    setIsError(false);
+    try {
+      const serverResp = await getPeople();
```

Comment:
Rename the variable `serverResp` to `serverResponse` to improve code readability 📚.